zerver/openapi/zulip.yaml

```diff
@@ -8420,6 +8420,29 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+    delete:
+      operationId: delete-custom-emoji
+      summary: Delete custom emoji
+      tags: ["server_and_organizations"]
+      description: |
+        This endpoint is used to delete a custom emoji from the user's
```

Comment:
"deactivate a custom emoji" here should be a Markdown link to the /help/ article for custom emoji noted above.